test/storage-test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
This is a good start, you'll need to make sure to test everything specified by the assignment and make sure the tests are running properly when you resubmit.